### PR TITLE
add support for lazy.nvim

### PR DESCRIPTION
**Pull request by saidelike**

### CODE REVIEW

File: apps/lazy.py [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+    def lazy_debug():
+        actions.user.vim_run_normal_exterm(":Lazy debug\n")
+
+    def lazy_help():
+        actions.user.vim_run_normal_exterm(":Lazy help\n")
+
+    def lazy_health():
+        actions.user.vim_run_normal_exterm(":Lazy health\n")
+
+    # def lazy_close():
```
Comment (fidgetingbits) [inline apps/lazy.py:107]: not sure if these commented out ones are from code you took from my talon-vim repo, but I guess we can remove them?

File: apps/lazy.py [hunk 2]
```diff
@@ -0,0 +1,150 @@
...
+    def lazy_update():
+        actions.user.vim_run_normal("U")
+
+    def lazy_sync():
+        actions.user.vim_run_normal("S")
+
+    def lazy_clean():
+        actions.user.vim_run_normal("X")
+
+    # def lazy_clear():
```
Comment (fidgetingbits) [inline apps/lazy.py:127]: here too

File: apps/lazy.py [hunk 3]
```diff
@@ -0,0 +1,150 @@
...
+    def lazy_profile():
+        actions.user.vim_run_normal("P")
+
+    def lazy_debug():
+        actions.user.vim_run_normal("D")
+
+    def lazy_help():
+        actions.user.vim_run_normal("?")
+
+    # def lazy_health():
```
Comment (fidgetingbits) [inline apps/lazy.py:147]: here too

**Review by fidgetingbits — APPROVED** [review]
approved in case you don't want to remove the comments for now, but won't merge in case you do

**Comment by saidelike** [comment]
> approved in case you don't want to remove the comments for now, but won't merge in case you do

In your original talon-vim repo, you had different commands in both lazy menu shown or not, and you defined them both in different talon files. I define the commands only once and have different implementations depending on case.

The reason they are commented is because they are not implemented in one of the case. But ya we could remove the comments if you prefer

**Comment by fidgetingbits** [comment]
ok if it's meant to be a marker to implement later can keep it then

**Comment by saidelike** [comment]
do we want to wait that we support detecting which nvim plugin is installed before merging this or not?
I am otherwise happy with this pr

**Comment by saidelike** [comment]
we agreed to put the plugins:
* into `core/plugins`: lazy.nvim
* into `core/tabs/{tabs.py/talon}`: taboo

**Comment by saidelike** [comment]
@fidgetingbits I ended up doing the updates we talked about. I will let you approve/merge if you are happy about it

File: core/plugins/lazy/lazy.py [hunk 4]
```diff
@@ -0,0 +1,150 @@
...
+        """open lazy home menu"""
+
+    def lazy_install():
+        """install missing lazy plugins"""
+
+    def lazy_update():
+        """update lazy plugins"""
+
+    def lazy_sync():
+        """run lazy install, clean and update"""
```
Comment (fidgetingbits) [inline core/plugins/lazy/lazy.py:33]: Something seems off with this line? Not sure if it would actually error

**Review by fidgetingbits — CHANGES_REQUESTED** [review]

File: core/plugins/lazy/lazy.py (hunk 4 above)
Comment (saidelike) [reply core/plugins/lazy/lazy.py:33]: Can you explain what seems off?

Comment (fidgetingbits) [reply core/plugins/lazy/lazy.py:33]: Seems like it's just a visual rendering bug on mobile, but the docstring shows up on a new line. I'm guessing if you don't see it, it's actually fine so will accept

**Review by fidgetingbits — APPROVED** [review]

File: core/plugins/lazy/lazy.py (hunk 4 above)
Comment (saidelike) [reply core/plugins/lazy/lazy.py:33]: Ya is probably that :)

**Pull request merged**